### ISSUE TITLE
Не работает изменение ширины горизонтальной линии

### ISSUE THREAD

**Issue by Fobos25** [SBProX/SBProX-Dev#1233]
При создании или изменении линии, меняю значение поля "Ширина", но ничего не происходит. Хотелось бы чтобы она формировала область, как в других терминалах. С чёткими линиями по краям, и полупрозрачной заливкой между ними.

**Comment by SBProX** [comment]
Для этого есть инструмент Range.
Посмотрите пожалуйста:

![image](https://user-images.githubusercontent.com/16720120/84650503-0c118100-af11-11ea-9148-a3ead2e34d5e.png)


Так же пожалуйста, высылайте сразу скриншоты.
Линий в платформе много, в каких конкретно настройках Вы меняете ширину не понятно.

**Comment by Fobos25** [comment]
![изображение](https://user-images.githubusercontent.com/53903337/85391749-41425280-b553-11ea-82c3-909d492c113f.png)
Да. Я его не заметил. Но.. Попробовал и диапазон закрывает основной график, он не прозрачный как у Вас на картинке и границы его показывают середину, а не верхнее и нижнее значение, как на Вашей картинке. Прошу сделать как на Вашем скрине

**Comment by Fobos25** [comment]
![изображение](https://user-images.githubusercontent.com/53903337/85392789-b3676700-b554-11ea-8e6c-7c43f95d5178.png)

**Comment by Fobos25** [comment]
> 
> 
> Для этого есть инструмент Range.
> Посмотрите пожалуйста:
> 
> ![image](https://user-images.githubusercontent.com/16720120/84650503-0c118100-af11-11ea-9148-a3ead2e34d5e.png)
> 
> Так же пожалуйста, высылайте сразу скриншоты.
> Линий в платформе много, в каких конкретно настройках Вы меняете ширину не понятно.

С диапазоном разобрался. Спасибо

**Comment by Fobos25** [comment]
Да. Диапазоны очень удобный инструмент, но к сожалению, он не линкуется. Прошу добавить линковку для диапазонов
